demos/tutorial/alignment/scoring_scheme_simple.cpp

```diff
@@ -0,0 +1,33 @@
+//![main]
+#include <iostream>
+#include <seqan/align.h>
+
+using namespace seqan;
+
+int main()
+{
+    typedef String<AminoAcid> TSequence;                 // sequence type
+    typedef Align<TSequence, ArrayGaps> TAlign;      // align type
```

Comment:
I have to say ... since it's a tutorial.
http://seqan.readthedocs.org/en/master/StyleGuide/Cpp.html#inline-comments